src/js/main.js

```diff
@@ -42,12 +48,38 @@ class Site {
   }
 
   initCoverVid() {
-    const _this = this;
-
-    if (_this.$covervidVideo.length) {
-      _this.$covervidVideo.coverVid(1920, 1080);
+    if (this.$covervidVideo.length) {
+      this.$covervidVideo.coverVid(1920, 1080);
     }
   }
+
+  bindStickyHeader() {
+    const that = this;
+
+    this.sizeHeaderSpacer();
+
+    this.$window.on('scroll', function() {
```

Comment:
The function that handles the scroll should be a different function (ex. `Site.handleOnScroll()`) that in the `constructor` is bound to `this` (ex. `this.handleOnScroll = this.handleOnScroll.bind(this)`. The same way we do on our React code.

Reply:
related discussion about this new class pattern https://github.com/facebook/react/issues/9851